test/e2e/helpers/bpmn-page-utils.ts

```diff
@@ -102,7 +101,7 @@ export class PageTester {
   }
 
   protected async doLoadBPMNDiagramInRefreshedPage(url: string, checkResponseStatus = true): Promise<void> {
-    const response = await this.page.goto(url);
+    const response = await this.page.goto(url, { waitUntil: 'load' });
```

Comment:
No need. `load` is the default value of the option `waitUntil` for the function `goto`: https://playwright.dev/docs/api/class-page#page-goto

```suggestion
    const response = await this.page.goto(url);
```

Reply:
done in e0bfe22